ConnectSecure/Deploy-ConnectSecure-V4.ps1

```diff
@@ -95,6 +96,12 @@ if([string]::IsNullOrEmpty($TenantID) -or [string]::IsNullOrWhiteSpace($TenantID
     $TenantID = '000000000000000000'    # Numeric Tenant ID string from the deployment script
 }
 
```

Comment:
Would be good to add a comment before this section with link to documentation at https://cybercns.atlassian.net/wiki/spaces/CVB/pages/2111242438/How+To+Install+V4+Agent+Using+RMM+Script for user reference.

Reply:
Added